integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java

```diff
@@ -299,6 +299,52 @@ public static Domain createDomainResource(
     return domainCR;
   }
 
+  /**
+   * Create a domain object for a Kubernetes domain custom resource using the basic WLS image and MII auxiliary image
+   * image.
+   *
+   * @param domainResourceName name of the domain resource
+   * @param domNamespace Kubernetes namespace that the domain is hosted
+   * @param baseImageName name of the base image to use
+   * @param adminSecretName name of the new WebLogic admin credentials secret
+   * @param repoSecretName name of the secret for pulling the WebLogic image
+   * @param encryptionSecretName name of the secret used to encrypt the models
+   * @param replicaCount number of managed servers to start
+   * @param clusterName name of the cluster to add in domain
+   * @param auxiliaryImageVolumes list of AuxiliaryImageVolumes
+   * @param auxiliaryImages list of AuxiliaryImages
+   * @return domain object of the domain resource
+   */
+  public static Domain createDomainResource(
```

Comment:
Can you rename the method name to createDomainResourceWithAuxiliaryImage()

Reply:
fixed